prioritization_fees/src/account_priofees.rs

```diff
@@ -23,37 +26,67 @@ pub struct AccountPrioStore {
     pub account_by_prio_fees_writeonly: Arc<DashMap<Pubkey, AccountPrio>>,
     pub number_of_slots_to_save: usize,
     pub last_slot: Arc<AtomicU64>,
+    pub address_lookup_tables_impl: Option<Arc<dyn AddressLookupTableInterface>>,
 }
 
 impl AccountPrioStore {
-    pub fn new(number_of_slots_to_save: usize) -> Self {
+    pub fn new(
+        number_of_slots_to_save: usize,
+        address_lookup_tables_impl: Option<Arc<dyn AddressLookupTableInterface>>,
+    ) -> Self {
         Self {
             account_by_prio_fees_all: Arc::new(DashMap::new()),
             account_by_prio_fees_writeonly: Arc::new(DashMap::new()),
             number_of_slots_to_save,
             last_slot: Arc::new(AtomicU64::new(0)),
+            address_lookup_tables_impl,
         }
     }
 
-    pub fn update(&self, produced_block: &ProducedBlock) -> AccountPrioFeesUpdateMessage {
+    pub async fn update(&self, produced_block: &ProducedBlock) -> AccountPrioFeesUpdateMessage {
         // sort by ascending order
```

Comment:
there's a `.rev()` further down; doesn't that change the sort order to _descending_?